github_pr.py

```diff
@@ -59,7 +59,7 @@ def _print_pr(pr, **args):
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
+        print "#%d [%s] %10s:%s <- %s:%-30s    %s" % (pr.number, pr.state, pr.base.repo.owner.login, pr.base.ref, pr.head.repo.owner.login, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
```

Comment:
I tried that string initially, and it lost the spacing - switching it to what I have now preserved the spacing (see output below - only sanitizing I did was renaming branches/forks)